runtime/compiler/optimizer/InterpreterEmulator.hpp

```diff
@@ -224,8 +226,10 @@ class InterpreterEmulator : public TR_ByteCodeIteratorWithState<TR_J9ByteCode, J
       void initializeIteratorWithState();
       /*
        * push and pop operands on stack according to given bytecode
+       *
+       * \return whether maintainStack succeed. Return false if there is any unexpected bytecode.
```

Comment:
would false be better as meaning some error occurred - eg an unexpected bytecode?

Reply:
modified the documentation msg.